config/locales/pl.yml

```diff
@@ -30,7 +30,8 @@ pl:
       alert: 'Nie masz tu dostępu'
     setting_controller:
       flash_message:
-        alert-info: 'Twoje ustawienia zostały zaktualizowane.'
+      update:
```

Comment:
You have different translations in pl and en files missing `alert-info` in English version.